README.md

```diff
@@ -17,19 +17,19 @@ func main() {
 
     token, err := client.GetToken("username", "password")
     if err != nil {
-        fmt.Println("Failed to obtain API token: %v", err)
+        fmt.Println("Failed to obtain API token", err)
```

Comment:
I think we still need the colon here or else the message is somewhat unreadable: 
```suggestion
        fmt.Println("Failed to obtain API token:", err)
```